extension/resources/walkthrough/setup.md

```diff
@@ -0,0 +1,16 @@
+# Setup Page
```

Comment:
Currently, the section's contents are pretty simple, but we can expand further. For example, highlighting each section. Any suggestions for improving the text are welcome!